src/App.jsx

```diff
@@ -14,7 +13,7 @@ function App() {
         <Route path="/" element={<WelcomePage />} />
         <Route path="/articles" element={<Articles />} />
         <Route path="/articles/:id" element={<ViewArticle />} />
-        <Route path="/articles/:id/comments" element={<Comments />} />
+        <Route path="/articles/:id/comments" element={<ViewArticle />} />
```

Comment:
Don't really need a separate route for comments if you're rendering the same component (routes don't need to match up with the backend api exactly). Up to you but if you just want to click a button to toggle viewing the comments component something like this might serve you better https://sentry.io/answers/how-do-you-show-or-hide-elements-in-react/